sass/sass-site
# [Playground] Debug links

## Description
In the debug console, line numbers are now links to select or position the cursor at the relevant issue. Also, deprecation messages with links are now clickable.

## Related Issue(s)
Addresses number 6 in #779.

## Steps to test/reproduce

1. [Debug, Warn, and Deprecation links](https://deploy-preview-1179--sass-lang.netlify.app/playground/#eJwzNHRISU0qTVfQMDQqqNA3KajQtOYCAEPTBfY=?s=L1C9-L1C17)
2.[ Error link](https://deploy-preview-1179--sass-lang.netlify.app/playground/#eJwzNFRJz89PsVJISkyx5uJKrOZSUEjOz8kvslJQKUoFCtUCAKdZCcA=?s=L4C10-L4C14)
3. [Error without a range](https://deploy-preview-1179--sass-lang.netlify.app/playground/#eJwzNExJTUkqTVcAAA4tAu4=)
4. [Long content, more examples](https://deploy-preview-1179--sass-lang.netlify.app/playground/#eJztls1qhDAUhfd5isuMLQr+JU4pKIXprq8RNaNhUg2JM4yUefdGHYvSdRfFbEK4OSfnfjebYHy8aAY7TbVOBdfdLkOLStGKVpkSOpYsv1TgYiJv0UHePFObi6MopELW1N0zXL6WxMsgigCvFa2kBe96V1W5S3DsAyEvw4J9iMODN3nMbnZNNz5cbySeBL+qphHn1DZdoDtanFP4YOLKOl5QH94VpyJDjlT8k6o+GPtIYZ8kiek/b8sevhDAyj4MIaRSsqZ0Fwc+aNroQDPFTwYeYDhKl9YM3RGi44WPnHWuSQR4Tuv2ytQg+pFN09EFFcxdW3xwBK/qrmHmLYDET2PyfQiyyBbZIltki2yRLbJFtsgW+U+R51/5qgGEyCpfsTLb6Hz+dfMbfC+LvAXkLTJb5C0gfwM06Ea3)

Links should allow you to open in new tab/window with CMD/Shift/Right click, etc.
